qipr_approver/approver/forms/about_you_form.py

```diff
@@ -31,10 +31,11 @@ def __init__(self, user=None, person=None):
                       'input_classes': 'about__field--box'}
 
         self.department = {'name': 'department',
-                           'placeholder': 'e.g. Pediatrics',
+                           'placeholder': 'Department',
                            'label': 'What is your primary department?',
-                           'type': 'text',
-                           'value': person.department or '',
+                           'selected': getattr(person.department,'name',''),
+                           'other': person.other_self_classification or '',
```

Comment:
This other field is if there should be a "other" option which would be filled in with text by the user. If we need a "other", then we need to add a "other_department" or whatever in the model. As this line stands, it is looking for other in self classification